source/quickstart/container.rst

```diff
@@ -6,35 +6,158 @@ Quickstart for Containers
 
 .. default-domain:: minio
 
-This page deploys MinIO as a container for use with early development
-and evaluation environments.
+This page deploys a standalone MinIO deployment as a as a container with an attached persistent volume for use with early development and evaluation environments.
 
-For instructions on deploying to production environments,
-see :ref:`deploy-minio-distributed`.
+For instructions on deploying to production environments, see :ref:`deploy-minio-distributed`.
 
 Prerequisites
 -------------
 
-ToDo
+- `Podman <https://podman.io/getting-started/installation.html>`_ or `Docker <https://docs.docker.com/get-docker/>`_ installed
+- Read, write, and delete access to the folder or drive used for the persistent volume
 
 Procedure
```

Comment:
Lets not lose the connectivity step. For each H2 we can still do numbered steps for:

1. Start the container
2. Connect your Browser
3. (Optional) Connect mc

Strictly speaking you can run `mc` with a container but it does not work well in my experience. We'll have to walk them through downloading and running mc